packages/api/src/external/commonwell/patient.ts

```diff
@@ -436,13 +443,50 @@ async function updatePatientAndLinksInCw(
       },
     });
 
-    await queryDocsIfScheduled(patient, getOrgIdExcludeList);
+    let foundNewDemographics = false;
```

Comment:
The logic below seems pretty similar to whats on create, could we make it reusable?

Reply:
Moved it to a helper function.

Reply:
Done (again).